scripts/index.js

```diff
@@ -0,0 +1,315 @@
+            <li>
+                <a href="#" class="animal">
+                    <img src="./icons/tasmanian-devil.png" alt="tasmanian-devil  Icon"
+                        class="icon-side tasmanian-devil-icon">
+                    <span class="nav-item">Tasmanian Devil</span>
+                </a>
+                <span class="tooltip">Tasmanian Devil</span>
+            </li>
+            <li>
+                <a href="#" class="animal">
+                    <img src="./icons/quokka.png" alt="quokka Icon" class="icon-side quokka-icon">
+                    <span class="nav-item">Quokka</span>
+                </a>
+                <span class="tooltip">Quokka</span>
+            </li>
+            <li>
+                <a href="#" class="animal">
+                    <img src="./icons/Frill-necked-lizard.png" alt="frill-necked-lizard Icon"
+                        class="icon-side frill-necked-lizard-icon">
+                    <span class="nav-item">Frill-Necked Lizard</span>
+                </a>
+                <span class="tooltip">Frill-Necked Lizard</span>
+            </li>
+            <li>
+                <a href="#" class="animal">
+                    <img src="./icons/hawksbill-turtle.png" alt="hawksbill-turtle Icon"
+                        class="icon-side hawksbill-turtle-icon">
+                    <span class="nav-item">Hawksbill Turtle</span>
+                </a>
+                <span class="tooltip">Hawksbill Turtle</span>
+            </li>
+            <li>
+                <a href="#" class="animal">
+                    <img src="./icons/monitor-lizard.png" alt="perentie  Icon" class="icon-side perentie-icon">
+                    <span class="nav-item perentie-klick">Perentie</span>
+                </a>
+                <span class="tooltip">Perentie</span>
+            </li>
+            <li>
+                <div id="perentie-info"></div>
+
+                <a href="#" class="animal">
+                    <img src="./icons/cassowary.png" alt="cassowary Icon" class="icon-side cassowary-icon">
+                    <span class="nav-item">Cassowary</span>
+                </a>
+                <span class="tooltip">Cassowary</span>
+            </li>
+            <li>
+                <a href="#" class="animal">
+                    <img src="./icons/kookaburra.png" alt="kookaburra Icon" class="icon-side kookaburra-icon">
+                    <span class="nav-item">Kookaburra</span>
+                </a>
+                <span class="tooltip">Kookaburra</span>
+            </li>
+            <li>
+                <a href="#" class="animal">
+                    <img src="./icons/cockatoo.png" alt="yellow-tailed-black-cockatoo Icon"
+                        class="icon-side eyellow-tailed-black-cockatoo-icon">
+                    <span class="nav-item">Yellow-Tailed Black Cockatoo</span>
+                </a>
+                <span class="tooltip">Yellow-Tailed Black Cockatoo</span>
+            </li>
```

Comment:
You should use a loop to generate the sidebar, employing the objects in the array.

Reply:
Fixed it